### ISSUE TITLE
Items identified after release v3.0.0

### ISSUE THREAD

**Issue by Don-Isdale** [plantinformatics/pretzel#416]
#### Issues reported 2024Oct 23 - 25, after v3.0.0 release on Oct23

- [x] showing handsontable license message : tables are displayed
 8e467905 Don't show Genotype Table if handsOnTableLicenseKey is not defined
 e2af2e9f Don't show Feature Table if handsOnTableLicenseKey is not defined

- [x] #415

- [x] Feature Search with two different types of features, and click on the green plus symbols from the search, only the first feature shows up as an arrow.  Example search names :
```
AVRIG14900
TraesCS1A03G0717000
```
7ce4a588 Update Feature Search results triangles which span multiple blocks

Tested OK, 2024Nov14 :

![Screenshot from 2024-11-14 19-54-09](https://github.com/user-attachments/assets/e55a18c2-5f38-49ee-adf3-30c9037d3fb2)

---

- [x] handle and report to GUI when upload VCF worksheet if the dataset name in the VCF sheet doesn't have a corresponding directory in the VCF directory on the server.
Currently this causes this error in the server log, and the server exits :
```
+ cd <$vcfDir>/Wheat_pangenomes_IWGSC_v2.1
/app/lb3app/scripts/vcfGenotypeLookup.bash: line 513: cd: <$vcfDir>/Wheat_pangenomes_IWGSC_v2.1: No such file or directory
+ echo Error: 0 'VCF dataset dir is not configured,' Wheat_pangenomes_IWGSC_v2.1, PWD=/
```

Changes implemented :

 8e75232f data_templates : copy new worksheet types from xltx to ots
 a021759e handle and report to GUI when error in upload VCF worksheet
 4891a951 Add server trace of unhandled exception and promise rejection

Tested OK 2024Nov18 :
![Screenshot from 2024-11-18 09-20-39](https://github.com/user-attachments/assets/cb077fae-aa08-40fc-824a-ed671a458a3e)

---

**Comment by Don-Isdale** [comment]
#### Genotype Table : Change highlight of selected SNPs

Reported issue (MS Teams, 2024Nov01 16:44) : 
"re. selected SNPs for genotype sample ordering : can it not be solid green, but rather transparent? at the moment it obscures the allele ... can it just be an outline ?"
In chrome the selected SNP highlight is showing as a solid green block;  in Firefox it shows a wider border at the top and bottom, allowing the allele (genotype value) of the cell to show through.

Firefox :

![Screenshot from 2024-11-07 10-14-08](https://github.com/user-attachments/assets/964d4956-d59c-4214-ac21-9330d2daf61b)

Change made :

+ change selected SNP highlight display - it is obscuring the genotype value in Chrome.
680b217f Genotype Table selected SNPs alt/ref : don't show the whole cell green in Chrome

**Comment by Don-Isdale** [comment]
#### Logout routes to /undefined and displays 404 page

When the user logs out from Pretzel, the URL changes to .../undefined, and the 404 error page is displayed.

**Comment by Don-Isdale** [comment]
#### Genotype Table : make selected SNPs consistent between datasets when a VCF dataset is viewed

- [x] retain or reset selected SNPs when another dataset is added in Genotype Table
efac7f4a retain selected SNPs when another dataset is added in Genotype Table

(This item was moved here from above [comment](https://github.com/plantinformatics/pretzel/issues/416#issuecomment-2460987048) 2024Nov07)

**Comment by Don-Isdale** [comment]
#### Column header height is too large for some datasets

+ [x] fix column header height :
69548904 exclude datasetId from Genotype Table column header height calculation

**Comment by Don-Isdale** [comment]
#### Closing as completed

All items are complete.

The implemented changes are released via PR #418 as v3.1.0.
